src/components/toolbar/tile-toolbar-button.tsx

```diff
@@ -0,0 +1,26 @@
+import classNames from "classnames";
+import React from "react";
+
+export interface TileToolbarButtonProps {
+  Icon: React.FC<React.SVGProps<SVGSVGElement>>;
+  onClick: (e: React.MouseEvent) => void;
+  selected?: boolean; // if undefined, considered to be false
+  enabled?: boolean;  // if undefined, considered to be true!
```

Comment:
It can cause tricky problems when something is considered true when undefined, so I'd suggest changing this property name to be `disabled`.  Slack post: https://concord-consortium.slack.com/archives/C0303QUNU/p1664938148115039